registry.json

```
@@ -1,4 +1,18 @@
 [
+
+   {
+       "ticker": "FLOW",                                                       
+       "poolId": "58920efd533bc451664937d03c14176b0693a1800326933f3abbc6e8",  
+       "social": {              
+         "website":"https://theriver906928134.wordpress.com", 
+         "twitter": "the_FLOW_pool",
+         "facebook": " ",  
+         "instagram": " ",
+         "youtube": " "
+       },
+       "telegram": "the_FLOW_pool",                                             
+       "operatorName": "Peer",                                  
+
```

You seem to be missing a brace:

```suggestion
},
```